Src/Support/Google.Apis.Auth/OAuth2/GoogleCredential.cs

```diff
@@ -33,16 +33,16 @@ namespace Google.Apis.Auth.OAuth2
     /// See <see cref="GetApplicationDefaultAsync(CancellationToken)"/> for the credential retrieval logic.
     /// </para>
     /// </summary>
-    public class GoogleCredential : ICredential, ITokenAccessWithHeaders, IOidcTokenProvider
+    public class GoogleCredential : IOidcTokenProvider, IAcceptsQuotaProject
     {
         /// <summary>Provider implements the logic for creating the application default credential.</summary>
         private static DefaultCredentialProvider defaultCredentialProvider = new DefaultCredentialProvider();
 
         /// <summary>The underlying credential being wrapped by this object.</summary>
-        protected readonly ICredential credential;
+        protected readonly IAcceptsQuotaProject credential;
```

Comment:
This would normally be a breaking change (albeit a corner case one) but I'm assuming that the internal constructor is the only available one, therefore there can't be subclasses outside the project, making it non-breaking?

Reply:
Yes, exactly only one internal constructor, so no one inhering here except us.